(FEATURE) - 18-game-score-holder

kevinboogaard opened the pull request:
https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/40

Als speler wil ik kunnen zien wat de huidige scorestand is zodat ik weet of ik aan het winnen ben.

kevinboogaard committed:
(FEATURE)-16-framework-timer

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/38

the framework now has a timer for the game

kevinboogaard committed:
(FEATURE)-17 -
framework-stopwatch

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/39

the framework now has a stopwatch for the game

kevinboogaard committed:
(FEATURE) -
16-framework-timer

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/38

the code is now commented

AlexAntonides committed:
(BUG) - 11-bug-animations
https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/33

kevinboogaard committed:
(FEATURE) -
18-game-score-holder

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/40

the game now has a scoreholder for the game score and the highscore

AlexAntonides committed:
(BUG) - 15-bug-alignment

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/37
Changed the way how the system works with reverting and such. Need to clean up the revert state. I found out that after falling the balls will stay in 'fall' state. Need to fix that later on. Creating issue now.

AlexAntonides committed:
(FEATURE) - 11-bug-animations

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/33
Added Swipe Wrong animation.

kevinboogaard committed:
(FEATURE) -
19-game-score-view

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/41

score view in the game also update score with a debug

kevinboogaard committed:
(FIX) -
18-game-score-holder

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/40

fix name

AlexAntonides committed:
(ENHANCEMENT) 14-enhancement-mobile-lag

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/36

Reduced image sizes.

AlexAntonides committed:
(BUG) - 11-bug-animations

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/33

Apparently the placeholder ball assets were still in the images.json. So it loaded the sprite and used it as an animation- so it didn't have any frames.

AlexAntonides committed:
(ENHANCEMENT) - 14-enhancement-mobile-lag

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/36

Changed the deltatime for both desktop and mobile. Cleaned up a bit.

AlexAntonides committed:
(ENHANCEMENT) - 14-enhancement-mobile-lag

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/36
Removed useless files.

AlexAntonides committed:
(FEATURE) - 20-game-input-computer

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/42
Added Swipe functionality. Changed on input down and up from game scene to tap. Now it's sort of working?

AlexAntonides committed:
(BUG) - 20-game-input-computer

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/42

kevinboogaard committed:
(FEATURE) - 23-game-coach-model

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/45

the game now has a task system

AlexAntonides reviewed (changes requested):
Needs the updated comment system before I will merge.

AlexAntonides committed:
Merge remote-tracking branch 'origin/23-game-coach-model' into 24-game-task-handler

AlexAntonides committed:
(FEATURE) 24-game-task-handler

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/46
Added the base files and comments.

AlexAntonides committed:
Merge branch '24-game-task-handler' into 16-framework-timer

AlexAntonides committed:
Merge branch '24-game-task-handler' into 17-framework-stopwatch

AlexAntonides committed:
Merge branch '16-framework-timer' into 24-game-task-handler

AlexAntonides committed:
(FEATURE) 24-game-task-handler

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/46
Added the base for the taskhandler / coachmodel.

AlexAntonides committed:
(FEATURE) 24-game-task-handler

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/46
Finished the task system.

AlexAntonides committed:
(REFACTOR) 24-game-task-handler

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/46
Removed the coach in the entityfactory.

kevinboogaard committed:
(FEATURE) 24-game-task-handler

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/46
Added the coach view in the game.

kevinboogaard committed:
(FEATURE) 24-game-task-handler

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/46
Added the base for the taskhandler / coachmodel.

kevinboogaard committed:
Merge branch '24-game-task-handler' of https://github.com/kevinboogaard/Sports_World-Ball_Pit into 24-game-task-handler

AlexAntonides committed:
(FEATURE) 29-game-time

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/57
Made a game time class and changed some utilities.

AlexAntonides committed:
Merge branch '24-game-task-handler' into 18-game-score-holder

AlexAntonides committed:
Merge remote-tracking branch 'origin/29-game-time' into 18-game-score-holder

AlexAntonides committed:
(ENHANCEMENT) 18-game-score

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/40
https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/41

Enhanced the view and the model so that it works with the interfacelayer. Also added the sprite.

kevinboogaard committed:
(FEATURE) 24-game-task-handler

https://github.com/kevinboogaard/Sports_World-Ball_Pit/issues/46
loaded all the spites

AlexAntonides committed:
Merge branch '24-game-task-handler' into 18-game-score-holder

# Conflicts:
#	advancedgames.ballpit.game/src/entities/views/coachview.js
